### DISCUSSION TITLE
emojis on starred list arent displayed at a correct size

### DISCUSSION THREAD

**Discussion by martimolanes** [community/community#47085] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/67336143/218275234-dfa8ce82-98a2-4ebc-8ca0-20bfc9d5ced5.png)
## Environment
Google Chrome Version 110.0.5481.77 (Official Build) (64-bit)
Ubuntu 22.04

**Comment by binharebs** [comment]
- [ ]

**Reply by binharebs** [reply]
@binharebs

**Reply by luutrutatcavaomaytinh** [reply]
Googlepro

**Reply by binharebs** [reply]
- [ ] - **_[@This was the first one ]()_**
